### PR TITLE
Fixes issue #7 whereby the read method should be renamed to raw_uv

### PR DESCRIPTION
**Pull request by WindFishDiety**
Addressing #7 
Updated code, comments and sample code.
Testing would be required

### CODE REVIEW

File: adafruit_veml6070.py [hunk 1]
```diff
@@ -145,18 +145,18 @@ def __init__(self, i2c_bus, _veml6070_it="VEML6070_1_T", ack=False):
 
 
     @property
-    def read(self):
+    def raw_uv(self):
```
Comment (caternuson) [inline adafruit_veml6070.py]: How about `uv_raw`? That'd be more inline with the VEML6075 that has `uva` and `uvb` properties.

File: adafruit_veml6070.py [hunk 2]
```diff
@@ -145,18 +145,18 @@ def __init__(self, i2c_bus, _veml6070_it="VEML6070_1_T", ack=False):
...
         """
         Reads and returns the value of the UV intensity.
         """
-        read_buf = bytearray(2)
+        uv_buf = bytearray(2)
```
Comment (caternuson) [inline adafruit_veml6070.py]: This buffer is just used locally and could be named anything. Naming if just `buf` or `buffer` might be better visually, so it doesn't get confused with the property name.

File: adafruit_veml6070.py [hunk 3]
```diff
@@ -145,18 +145,18 @@ def __init__(self, i2c_bus, _veml6070_it="VEML6070_1_T", ack=False):
...
         with self.i2c_low as i2c_low:
-            i2c_low.readinto(read_buf, end=1)
+            i2c_low.readinto(uv_buf, end=1)
 
         with self.i2c_high as i2c_high:
-            i2c_high.readinto(read_buf, start=1)
+            i2c_high.readinto(uv_buf, start=1)
 
-        uvi = read_buf[1] << 8 | read_buf[0]
+        uvi = uv_buf[1] << 8 | uv_buf[0]
```
Comment (caternuson) [inline adafruit_veml6070.py]: This variable doesn't get used, so could be combined on the return line:
```python
return uv_buf[1] << 8 | uv_buf[0]
```

**Review by caternuson — CHANGES_REQUESTED** [review]
Looks good. My suggestions are essentially cosmetic.

**Review by sommersoft — DISMISSED** [review]
Looks good. Thanks for the fix @WindFishDiety!

**Comment by WindFishDiety** [comment]
Hey thanks for letting me participate in the pycon sprints.

**Review by caternuson — APPROVED** [review]
Yep, thanks. Looks good!

**Pull request merged**